src/modules/coreinit/coreinit_exit.cpp

```diff
@@ -0,0 +1,24 @@
+#include "coreinit.h"
+#include "coreinit_exit.h"
+#include "utils/log.h"
+
+void
+_Exit()
```

Comment:
I won't say I'm actively involved in compiler development (since I'm not) but I don't know of any existing compilers that mangle names in a way that could cause collisions with library namespace, so IMO it's more useful to preserve the correlation with coreinit function names than to ensure absolute standards compliance -- particularly since this is a standalone project rather than a library, so even if a name change did become necessary it would have no external impact. But of course you can change the name if you want.